packages/contracts-periphery/package.json

```diff
@@ -63,7 +63,8 @@
     "@nomiclabs/hardhat-etherscan": "^3.0.3",
     "@nomiclabs/hardhat-waffle": "^2.0.3",
     "@rari-capital/solmate": "7.0.0-alpha.3",
-    "@openzeppelin/contracts": "4.6.0",
+    "@openzeppelin/contracts": "^4.8.0",
```

Comment:
Would prefer to pin to an exact version here so that upgrading is very intentional. We are using `4.7.3` in `contracts-bedrock` for both oz contracts packages, could we pin to that?